### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pwilson77**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+						</p>
+						<p class="left rec_caption">
+							The Odin Project <br>
+							10 views
+						</p>
+					</a>
+				</div>
+
+				<button class="btn show-more">show more</button>
+		</aside>
```
Comment (zlayabekrija) [inline index.html:274]: please don't forget to clear the floats after using them.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+.show-more{
+	width: 100%;
+	    border-color: transparent;
+}
+
+.d-head{
+	display: inline-block;
+}
+.subscribe{
```
Comment (zlayabekrija) [inline main.css]: this class is not in use, please delete it.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great work guys.
Minor things to change follow up.
`<a>` tags if they don't have a link, they should have the value of `href="#"`
`` should have alt attribute present.
And after fixing those things, submit a new review request.
Happy coding.

File: index.html (hunk 1 above)
Comment (truetechcode) [reply index.html:274]: All correction effected thanks.

**Pull request closed without merging**